modules/@apostrophecms/page/index.js

```diff
@@ -1675,6 +1675,24 @@ database.`);
       // Parks one page as found in the `park` option. Called by
       // `implementParkAll`.
       async implementParkOne(req, item) {
+        if (item.parkedId === 'trash') {
```

Comment:
This is new logic to cope with an edge case: project level developers currently have no choice but to redeclare the trashcan as a parked page if they want to change the type of the home page, and we did this in a3-assembly-boilerplate. It's alpha, so we can stop the show and tell them to update it to be an archive page, but we can't fail to communicate with a graceless mystery error, which was happening.

Reply:
I don't think this is how we should do this. We're baking in another thing that will be quickly deprecated while we're in alpha. If people are re-parking the home page and know enough that they need to re-park the "trash," they should know enough to look at the core implementation. They can then see the actual slug.

This is not some general error case we should provide and error for. It's an extremely specific error for A2 devs who aren't looking carefully at what has changed.

Reply:
This isn't for A2 devs who aren't looking carefully. This is for A3 alpha
devs who already looked carefully and figured out this was the only way
forward - including our own enterprise client.